utils/installer.py

```diff
@@ -258,17 +258,17 @@ def setup_libvirt():
 
 
 def setup_kubevirt():
-    master = deploy.kubevirt.master
-    master_username = deploy.kubevirt.master_user
-    master_password = deploy.kubevirt.master_passwd
+    master = deploy.kubevirt.endpoint
+    master_username = ''
+    master_password = deploy.kubevirt.token
     guest_name = deploy.kubevirt.guest_name
     guest_username = deploy.kubevirt.guest_user
     guest_password = deploy.kubevirt.guest_passwd
     ssh_master ={
```

Comment:
is it necessary to remain this dict?

Reply:
ssh_master is not needed, will remove it.